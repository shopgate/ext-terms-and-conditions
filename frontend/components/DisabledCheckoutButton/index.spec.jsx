import React from 'react';
import { mount } from 'enzyme';

describe('DisabledCheckoutButton', () => {
  it('should render', () => {
    // eslint-disable-next-line global-require
    const DisabledCheckoutButton = require('./index').default;
    const component = mount(<DisabledCheckoutButton />);
    expect(component).toMatchSnapshot();
    expect(component.html()).not.toBe(null);
  });
});
