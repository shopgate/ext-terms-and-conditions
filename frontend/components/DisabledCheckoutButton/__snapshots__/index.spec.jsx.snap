// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DisabledCheckoutButton should render 1`] = `
<DisabledCheckoutButton
  isActive={false}
  isOrderable={false}
  showTermsNotice={[Function]}
>
  <RippleButton
    className="css-ce5yef"
    disabled={false}
    flat={false}
    onClick={[Function]}
    rippleClassName=""
    rippleSize={null}
    testId="Button"
    type="regular"
    wrapContent={true}
  >
    <Button
      className="css-ce5yef"
      disabled={false}
      flat={false}
      onClick={[Function]}
      testId="Button"
      type="regular"
      wrapContent={false}
    >
      <Button
        className="css-kevfw8 css-ce5yef"
        disabled={false}
        onClick={[Function]}
        testId="Button"
      >
        <button
          className="css-kevfw8 css-ce5yef css-1rn65c0"
          data-test-id="Button"
          disabled={false}
          onClick={[Function]}
        >
          <Ripple
            className="css-1fscwua "
            color="#000"
            disabled={false}
            fill={true}
            onClick={[Function]}
            onComplete={[Function]}
            overflow={false}
            size={null}
          >
            <div
              aria-hidden={true}
              className="css-1fscwua "
              data-test-id="Ripple"
              onClick={[Function]}
              style={null}
            >
              <Translate
                className=""
                params={Object {}}
                string="cart.checkout"
              >
                <span
                  className=""
                >
                  cart.checkout
                </span>
              </Translate>
            </div>
          </Ripple>
        </button>
      </Button>
    </Button>
  </RippleButton>
</DisabledCheckoutButton>
`;
